.github/workflows/lock-closed-issues-workflow.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Lock threads'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  lock:
+    name: Lock closed issues
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          github-token: ${{ github.token }}
+          issue-lock-inactive-days: 30
+          pr-lock-inactive-days: 30
+          issue-lock-comment: >
+            This issue has been automatically locked since there
+            has not been any recent activity after it was closed.
+            Please open a new issue for related bugs.
+          pr-lock-comment: >
+            This pull request has been automatically locked since there
+            has not been any recent activity after it was closed.
+            Please open a new issue for related bugs.
```

Comment:
I would add the EOF newline. Prettier is checking that on ts files, but there are still non-ts files without it in the repos.

Reply:
Fixed.